datasets/acute-care/extraction_wrapper.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+"""
+Wrapper script to initiate a hail batch, and submit using an analysis-runner managed dataproc cluster
+"""
+import hailtop.batch as hb
+import os
+import sys
+
+from analysis_runner import dataproc
+
+
+def main():
+    """
+    Create a Hail Batch
+    Use the analysis-runner helper to generate a DataProc cluster, and add the job
+    Set off the batch
+    """
+
+    service_backend = hb.ServiceBackend(
+        billing_project=os.getenv('HAIL_BILLING_PROJECT'), bucket=os.getenv('HAIL_BUCKET')
+    )
+
+    # create a hail batch
+    batch = hb.Batch(name='cohort_mt_extraction', backend=service_backend)
+
+    my_cluster = dataproc.hail_dataproc_job(
```

Comment:
The function a job object, so might wanna rename `my_cluster` to `my_job` or something